### DISCUSSION TITLE
How can I get the 4.4.0 prisma version?

### DISCUSSION THREAD

**Discussion by Kalo0m** [prisma/prisma#15394] [Q&A] [answered]
Hi guys, I would like to have this bug fix: https://github.com/prisma/prisma-engines/pull/3198
How can I do that? I've seen there is a 4.4.0-dev.44 on npm. can I use it, if yes, how can I get it and if no, is there an alternative to get this bug fix?

**Comment by janpio** [comment, marked as answer]
4.4.0 will be release next week.
Until then you can in theory use `prisma@dev` and `@prisma/client@dev` from Npm - but that is not a stable version and might have other undesired changes. Please tread carefully.

**Reply by Kalo0m** [reply]
ok nice, Thanks a lot for your answer!
